app/views/pages/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="container_page">
+  <section class="main_section">
+    <div class="title">
+      <h1>Lets Learn</h1>
+    </div>
+
+    <div class="description">
+      <p> 
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+        nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in 
+        reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
```

Comment:
It would be good if you could post some content related to your project.

Reply:
Resolved